cmd/verify.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/dweepgogia/new-manifest-verification/pkg/validate"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(verifyCmd)
+}
+
+var verifyCmd = &cobra.Command{
+	Use:   "verify",
+	Short: "Validate YAML against OLM'c csv type.",
+	Long:  `Some long description about verify command`,
```

Comment:
Wrong long description. Btw is it the latest type? a specific version? Please explain this in the long description below.